labellab-flask/api/controllers/issuecontroller.py

```diff
@@ -125,13 +130,28 @@ class IssueInfo(MethodView):
     Url --> /api/v1/issue/issue_info/<int:project_id>/<int:issue_id>
     """
     @jwt_required
+    @project_member_only
     def get(self,project_id, issue_id):
         try:
-                        
+            if not issue_id:
+                response = {
+                    "success":False,
+                    "msg": "Issue id not provided"
+                }
+                return make_response(jsonify(response)), 400
+            
+            issue = find_by_id(issue_id)
+            if not issue:
```

Comment:
We're doing this check to see if the issue exists or not at 4 different places.
Maybe we can create a common decorator for it then?
If the issue with this id is not present, the decorator will throw the 404 response, else it will move onto the route method.

Reply:
Then we can reuse the middleware at other places also. Listing issues, comment endpoints etc

Reply:
Added the decorator. I will try to prevent code repetition in future.

Reply:
No worries. Thanks for the quick changes! 🎉